apps/next/components/Search.tsx

```diff
@@ -2,9 +2,17 @@ import React, { useState } from 'react'
 import { useRouter } from 'next/router'
 import Logo from '../public/Logo.png'
 import Search from '../public/Search.png'
+import Github from '../public/github.png'
```

Comment:
None of these imported logos `Github`, `Discord`, `Twitter`, etc. are used in this component and should ultimately only be required within the `Side_Navbar` or `Side_Navbar_Choice` component. I would eventually like to go through all files and remove unused, unnecessary, and redundant imports, but that can be a task for another pull request.